matRad_calcPhotonDose.m

```diff
@@ -197,21 +183,80 @@
     % limit rotated coordinates to positions where ray tracing is availabe
     rot_coordsV = rot_coordsV(radDepthIx,:);
     
+    % get index of central ray
+    for k = 1:stf(i).numOfRays
+        if (stf(i).ray(k).rayPos_bev(1) == 0).* (stf(i).ray(k).rayPos_bev(2) == 0).* ...
+           (stf(i).ray(k).rayPos_bev(3) == 0) == 1
+            center = k;
+        end
+    end
+    
```

Comment:
What happens if central ray not part of the ensemble of rays. This is well possible if the isocenter lies outside of the target!